corehq/apps/hqmedia/views.py

```diff
@@ -942,6 +942,8 @@ def _download_index_files(app, build_profile_id, is_retry=False):
         except ResourceConflict as e:
             if is_retry:
                 raise e
+            # Reload app and retry
+            app = get_app(app.domain, app.id)
```

Comment:
When this says "app", does it mean "app", or "build", or both? If its a build, then it might be interesting to get alerted on what has changed. I think that nothing should change except for comment on build? or perhaps there is some weirdness around build profiles being updated at the same time?